### PR TITLE
Disambiguate the glossary term "Safe"

### PR DESCRIPTION
**Pull request by hamishwillee**
Follows on from discussion here: https://github.com/mdn/content/pull/3562#issuecomment-809145430

Basically "Safe" is pretty unhelpful as a topic in the glossary for determining relevance.

### CODE REVIEW

File: files/en-us/glossary/safe/http/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Safe (HTTP Methods)
+slug: Glossary/safe/http
```
Comment (chrisdavidmills) [inline files/en-us/glossary/safe/http/index.html:3]: Just tested this, and it all looks fine to me, my only nitpick is that "Safe" in the slugs should have a capital "S" in both pages.

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Mostly good, just one comment to look at.

File: files/en-us/glossary/safe/http/index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+title: Safe (HTTP Methods)
+slug: Glossary/Safe/http
```
Comment (hamishwillee) [inline files/en-us/glossary/safe/http/index.html:3]: @chrisdavidmills Thanks. I've updated both the S in slugs. I assume HTTP is OK lower case.

Comment (chrisdavidmills) [reply files/en-us/glossary/safe/http/index.html]: lol, that's a good point ;-)

I've changed it to all upper case, as that's how it is normally written

**Review by chrisdavidmills — APPROVED** [review]

**Pull request merged**